week-1/interpret/time-format.js

```diff
@@ -9,18 +9,36 @@ const totalHours = (totalMinutes - remainingMinutes) / 60;
 const remainingHours = totalHours % 24;
 
 const result = `${remainingHours}:${remainingMinutes}:${remainingSeconds}`;
+
+// the line of code below added by me
+const result2AddedByMe = `${totalHours}:${remainingMinutes}:${remainingSeconds}`;
 console.log(result);
 
+console.log(result2AddedByMe);
+
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+// 7 variable decleration
 
 // b) How many function calls are there?
+//nothing , there/s not any function in this code .
 
 // c) Using documentation, explain what the expression movieLength % 60 represents
+//It shows the remaining value of the division of `movieLength` by 60. '%' is called the modulus 
+// arithmetic operator, which calculates the remainder of a division.
 
 // d) Interpret line 4, what does the expression assigned to totalMinutes mean?
+// line 4 is calculating number of minutes in the `movielength` . it subtract whole the seconds for remained seconds to get a dividable
+//number by 60 and give us back the whole minutes in the movieLength.
```

Comment:
Excellent explanation - I like your use of an example with a numerical calculation too

Reply:
because i understand numbers better than words lol